### ISSUE TITLE
Stopping playback

### ISSUE THREAD

**Issue by dw20155** [kodi-connect/kodi-alexa-video#2]
Hi,

I noticed that when you say 'Alexa Stop' then Kodi pauses playback as opposed to stops the video from playing.

Is that by design or a limitation of the video API?

fast forward and rewind don't seem to be working at the moment either.

**Comment by tkislan** [comment]
I have to resolve Stop command with Amazon. As they are sending `Pause` command for both Pause and Stop utterances.

Fast forwarding and rewinding is work in progress :)

**Comment by tkislan** [comment]
@dw20155 Can you please recheck with new version?
I've tried yesterday, and it actually stopped the playback .. so that I can close this issue

**Comment by dw20155** [comment]
tried again just now and it didnt work.

1. I download 0.18 kodi addon and then extracted to %appdata%/kodi/pluggins
2. launched kodi and filled out my email username in settings
4. got the connected messsage
5. Said "Alexa, play interstellar"
6. Interstellar starts playing
7. Said "Alexa Pause"
8. Movie is paused
9. Said "Alexa Play"
10. Movie resumes
11. said "Alexa Stop"
12. Movie pauses

from the debug log I can see:
`22:01:32.041 T:9664   DEBUG: [KodiConnect] {u'data': {u'commandType': u'pause', u'type': u'command'}, u'correlationId': u'37d9aa74-8664-49c6-9c65-caec445777ae'}
22:01:32.041 T:9664   DEBUG: [KodiConnect] handler data: {u'commandType': u'pause', u'type': u'command'}
22:01:32.041 T:9664   DEBUG: [KodiConnect] pause_handler
22:01:32.041 T:9664   DEBUG: [KodiConnect] handler response_data: {'status': 'ok'}`

**Comment by tkislan** [comment]
Okay, I'll have to create a bug on Amazon forums for this, because I've checked logs, and Alexa keeps sending `Pause` command anyway.

**Comment by tkislan** [comment]
@Malav20 how is this related to this issue?
Btw. https://github.com/kodi-connect/kodi-alexa-video#beta-testing

**Comment by newoski** [comment]
is this still actively being developed? any idea if the stop/pause bug if fixable?

**Comment by tkislan** [comment]
@newoski Yes, I still want to finish of some stuff, and then get it submitted.
Stop/pause bug is related to Amazon, and they are not very chatty on their developer forums.

**Comment by starmanrf** [comment]
Hi, not sure if this helps, but I am able to stop playback when I ask "Alexa, stop <device_name> , in my instance my device name is 'the TV' which I named on the Kodi connect device website to get the secret code for input in the Kodi addon .... Works like a charm for me.
